### PR TITLE
DM-40480: Slew Controller widget, improved Booster Valve control

### PR DESCRIPTION
**Pull request by pkubanek**
- SlewControllerPageWidget
- Updated doc strings.

### CODE REVIEW

File: python/lsst/ts/criopy/gui/ArrayGrid.py [hunk 1]
```diff
@@ -214,7 +218,28 @@ def data(self, data: typing.Any) -> None:
...
+    """Variable with values in array. Display array values to cells in
+    ArrayGrid.
+
+    Parameters
+    ----------
+    field : `str`
+        Signal field - member of the topics, holding the data.
+    label : `str`
+        Label for data, text describing data content and show before the data.
+    widget : UnitLabel, optional
```
Comment (teweitsai) [inline python/lsst/ts/criopy/gui/ArrayGrid.py:230]: UnitLabel -> `UnitLabel `

File: python/lsst/ts/criopy/gui/ArrayGrid.py [hunk 2]
```diff
@@ -423,10 +468,15 @@ class ArrayGrid(QWidget):
...
         equals to number of items/variables.
-
-    orientation : `Qt.Orientation`
+    items : `[AbstractColumn]`
+        Items displayed in the grid.
+    orientation : `Qt.Orientation`, optional
         Grid orientation. Vertical displays in columns indices and in rows
-        values. Horizontal swaps rows and columns.
+        values. Horizontal swaps rows and columns. Default to Vertical.
+    chart : TimeChart, optional
```
Comment (teweitsai) [inline python/lsst/ts/criopy/gui/ArrayGrid.py:476]: TimeChart -> `TimeChart`

File: python/lsst/ts/criopy/gui/ArrayGrid.py [hunk 3]
```diff
@@ -423,10 +468,15 @@ class ArrayGrid(QWidget):
...
+        Optional timechart associated with the grid. If provided, mouseclicks
+        on a data label in the grid will display timeserie of that label in the
```
Comment (teweitsai) [inline python/lsst/ts/criopy/gui/ArrayGrid.py:478]: timeserie -> timeseries

File: python/lsst/ts/criopy/gui/sal/StateEnabled.py [hunk 4]
```diff
@@ -34,13 +35,13 @@
...
-class SignalButton(QPushButton):
+class SignalButton(ColoredButton):
     """Push button enabled by signal.value in given state.
 
     Parameters
     ----------
-    title : `str`
-        Button title. Passed to QPushButton.
+    text : `str`
+        Button text. Passed to ColoredButton
```
Comment (teweitsai) [inline python/lsst/ts/criopy/gui/sal/StateEnabled.py:44]: Period in the end:

```
Passed to ColoredButton.
```

File: python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py [hunk 5]
```diff
@@ -62,7 +62,10 @@ def setState(self, state: int) -> None:
             return
 
         rangeRatio = 0.1
-        if state == DetailedState.RAISING or state == DetailedState.RAISINGENGINEERING:
+        if (
+            state == DetailedStates.RAISING
+            or state == DetailedStates.RAISINGENGINEERING
+        ):
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py:68]:
```
if state in (DetailedStates.RAISING, DetailedStates.RAISINGENGINEERING):
```

File: python/lsst/ts/criopy/m1m3/force_grid.py [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+
+class Forces(ArrayFields):
+    """Class to display all six forces and moments components in ArrayGrid.
+
+    Parameters
+    ----------
+    label : `str`
+    signal : Signal
+    extra_widgets : `[QWidget]`
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/force_grid.py:38]: Doc string?

Signal -> `Signal `

**Review by teweitsai — APPROVED** [review]
Looks good!

File: python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py (hunk 5 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py:68]: Thanks.

**Pull request merged**